livekit-plugins/livekit-plugins-cartesia/livekit/plugins/cartesia/tts.py

```diff
@@ -263,16 +240,16 @@ async def recv_task():
                             )
                         )
 
-                    pending_segments.remove(segment_id)
-                    if len(pending_segments) == 0 and self._input_ch.closed:
+                    if segment_id == segment_id:
```

Comment:
this is always true?

Reply:
I think you still need a list of pending segments (There could be multiple flush and the audio is always late)

Reply:
woops yes, funny because it actually might not matter that we go the segment id check because only one is used per connection right?

Reply:
Does the old code actually do this? Multiple context_ids actually causes issues. Cartesia runs them in parallel resulting in overlapping audio